examples/prophet/MLproject

```diff
@@ -0,0 +1,10 @@
+name: tutorial
+
+conda_env: conda.yaml
+
+entry_points:
+  main:
+    parameters:
+      csv_url: {type: uri, default: 'https://raw.githubusercontent.com/facebook/prophet/master/examples/example_wp_log_peyton_manning.csv'}
```

Comment:
Any particular reason you both inline the csv and refer to it by URI? Probably either is fine, but may be good to be consistent. Also, if we use the URI, perhaps we could point to a stable one (commit based) in case they later move the file.

Reply:
good idea.  I was really just copying from the wine example but I'll pull it out of the code since it's better as input